src/blog/2024/03/using_kafka_with_node-red.md

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
@sumitshinde-84 Can we not use `_` that end up in any URL? We use `-` instead

Reply:
Okay zj, I will replace underscores with dashes.